### PR TITLE
post about EOL EL6,7 and customer portal

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: _posts/2020-27-11-eol.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title:  "EOL of EL6 and EL7 and removal Customer portal support"
+date:   2020-11-27 12:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+
+## Packages will not be built for EL6 and EL7.
+
+For [long](https://github.com/abrt/faf/issues/735) [time](https://github.com/abrt/abrt/issues/1337), we talked about EOLing EL6 and EL7. What it actually means?
```
Comment (mzidek-gh) [inline _posts/2020-27-11-eol.markdown]: What it actually means? -> What does it actually mean?

File: _posts/2020-27-11-eol.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+RHEL 6 is [going to be EOL](https://access.redhat.com/discussions/2399461) at the end of this month (30th November 2020). We will no longer build ABRT packages for EL6 and we will stop supporting EL6 as content.
+
+RHEL 7 is still active but quite old. ABRT team will stop testing, building, and developing on top of RHEL7. We are going to focus on RHEL 8 and upcoming RHEL and Fedora. We still support RHEL 7 as content (e.g., in ABRT Analytics).
+
+This is a description of the current de-facto state. The last release of ABRT Analytics was not already build for RHEL 7.
```
Comment (mzidek-gh) [inline _posts/2020-27-11-eol.markdown]: not already -> already not

File: _posts/2020-27-11-eol.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## End of reporting to Customer Portal
+
+Red Hat Customer Portal migrated to a new backend that does not have the API which we need and as result of that we removed reporting to Customer Portal from ABRT.
```
Comment (mzidek-gh) [inline _posts/2020-27-11-eol.markdown]: This is a small nitpick, but it was not removed directly from ABRT but from libreport. For example Anaconda that uses libreport directly will  also  be affected even if they do not use ABRT.

Comment (michalfabik) [inline _posts/2020-27-11-eol.markdown]: > as result of that  
  
"of that" isn't needed

File: _posts/2020-27-11-eol.markdown [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "EOL of EL6 and EL7 and removal Customer portal support"
+date:   2020-11-27 12:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+
+## Packages will not be built for EL6 and EL7.
```
Comment (michalfabik) [inline _posts/2020-27-11-eol.markdown]: will no longer be built

File: _posts/2020-27-11-eol.markdown (hunk 1 above)
Comment (michalfabik) [inline _posts/2020-27-11-eol.markdown]: For _a_ long time

File: _posts/2020-27-11-eol.markdown (hunk 3 above)
Comment (michalfabik) [inline _posts/2020-27-11-eol.markdown]: I'd say "_was_ migrated"

Comment (mzidek-gh) [reply _posts/2020-27-11-eol.markdown]: also "as result" should be "as a result" (with the article 'a')

**Comment by xsuchy** [comment]
Updated.

**Pull request merged**